### PR TITLE
docs: update SSH docs to add keys

### PR DESCRIPTION
**Pull request by runcom**
Following https://bugzilla.redhat.com/show_bug.cgi?id=1744940 it's clear that we're missing on documentation about how to add ssh keys. Fix that.

### CODE REVIEW

**Comment by runcom** [comment]
> /hold

added the 4.3 brackets in the title if that was what the hold was for :)

File: docs/Update-SSHKeys.md [hunk 1]
```diff
@@ -22,6 +22,37 @@ You will need the following information for the MachineConfig that will be used
 
 - `sshAuthorizedKeys:` you will need one or more public keys to be assigned to user `core`.  Multiple SSH Keys should begin on different lines and each be preceded by `-`.
 
+## Creating SSH for workers
+
+In case your cluster has been installed without providing the ssh key for the `core` user at installation time, you can easily create a MachineConfig that will ship the key to the nodes in a pool:
```
Comment (kikisdeliveryservice) [inline docs/Update-SSHKeys.md]: i'd add:
"installation time (verify that `99-worker-ssh`/`99-master-ssh` are missing, by checking `oc get machineconfigs`),

File: docs/Update-SSHKeys.md [hunk 2]
```diff
@@ -22,6 +22,37 @@ You will need the following information for the MachineConfig that will be used
...
+        - ssh-rsa ABC123....
+```
+
+Then just create the file with `oc`:
+
+```console
+$ oc create -f 99-worker-ssh.yaml
+```
+
+The MCO will take care of shipping the key to the nodes in the `worker` pool. The same procedure applies to any other pool in the cluster.
```
Comment (kikisdeliveryservice) [inline docs/Update-SSHKeys.md]: keys

File: docs/Update-SSHKeys.md [hunk 3]
```diff
@@ -22,6 +22,37 @@ You will need the following information for the MachineConfig that will be used
 
 - `sshAuthorizedKeys:` you will need one or more public keys to be assigned to user `core`.  Multiple SSH Keys should begin on different lines and each be preceded by `-`.
 
+## Creating SSH for workers
```
Comment (kikisdeliveryservice) [inline docs/Update-SSHKeys.md]: Creating SSH keys for a Pool

File: docs/Update-SSHKeys.md (hunk 1 above)
Comment (kikisdeliveryservice) [inline docs/Update-SSHKeys.md]: ssh keys

File: docs/Update-SSHKeys.md [hunk 4]
```diff
@@ -22,6 +22,37 @@ You will need the following information for the MachineConfig that will be used
...
+  name: 99-worker-ssh
+spec:
+  config:
+    ignition:
+      version: 2.2.0
+    passwd:
+      users:
+      - name: core
+        sshAuthorizedKeys:
+        - ssh-rsa ABC123....
```
Comment (kikisdeliveryservice) [inline docs/Update-SSHKeys.md]: add another fake key so it's clear you can add multiple

**Pull request merged**